README.md

```diff
@@ -1 +1,66 @@
-# Microverse-Tic-Tac-Toe-Ruby
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Tic Tac Toe
+
+In order to win the game, a player must place three of their marks in a horizontal, vertical, or diagonal row.
+Players will soon discover that the best play from both parties leads to a draw. 
+
+## How do we play Tic Tac Toe?
+
+- Run main on your terminal
+- The game is played on a grid that's 3 squares by 3 squares.
+- You are X, secound player is O. Players take turns putting their marks in empty squares.
```

Comment:
- Instead of `you are X` put `The first player is X`.
- You have a misspelling, you write `secound` it should be `second`, use Grammarly to check your grammar, an error on the grammar makes your repo looks unprofessional.